build-output-api/edge-functions/README.md

```diff
@@ -2,7 +2,7 @@
 
 ### Build Output API
 
-This Prebuilt Deployment example demonstrates how to output Vercel Edge Functions using the Build Output API.
+This Prebuilt Deployment example demonstrates how to output Vercel Edge Functions using the [Build Output API](https://vercel.com/docs/build-output-api/v3#vercel-primitives/edge-functions).
```

Comment:
For each of these links, I wasn't sure if it would work better to leave the label as `Build Output API` or if we should make it clearer that it links to a specific part, like `Build Output API :: Edge Functions`.